plugin-maven/src/test/java/com/diffplug/spotless/maven/kotlin/DiKTatTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2021 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.spotless.maven.kotlin;
+
+import com.diffplug.spotless.maven.MavenIntegrationHarness;
+
+import org.junit.Test;
+
+import java.io.File;
+
+public class DiKTatTest extends MavenIntegrationHarness {
+
+	@Test
+	public void testDiktat() throws Exception {
+		writePomWithKotlinSteps("<diktat/>");
+
+		String path1 = "src/main/kotlin/Main1.kt";
+		setFile(path1).toResource("kotlin/diktat/Main1.kt");
+		mavenRunner().withArguments("spotless:apply").runNoError();
+		assertFile(path1).sameAsResource("kotlin/diktat/Main1After.kt");
+
+	}
+
+	@Test
+	public void testDiktatShyiko() throws Exception {
+		writePomWithKotlinSteps("<diktat><version>0.4.0</version></diktat>");
+
+		String path1 = "src/main/kotlin/Main1.kt";
+		setFile(path1).toResource("kotlin/diktat/Main1.kt");
+		mavenRunner().withArguments("spotless:apply").runNoError();
+		assertFile(path1).sameAsResource("kotlin/diktat/Main1After.kt");
+	}
+
+	@Test
+	public void testDiktatConfig() throws Exception {
+
+		String configPath = (new File("").getAbsolutePath()) + "\\src\\main\\resources\\kotlin\\diktat\\diktat-analysis.yml";
```

Comment:
Use platform-independent paths. Also I believe more idiomatic way would be to use the same `toResource` utility methods